src/validations/loginValidation.js

```diff
@@ -0,0 +1,54 @@
+const inputValidation = require('./inputValidation');
+const UsersInteractor = require('../interactors/usersInteractor');
+const Encryptor = require('../utils/encryptor');
+
+class LoginValidation extends inputValidation {
+  constructor() {
+    super();
+    this.getLogger = this.getLogger.bind(this);
+    this.validateInput = this.validateInput.bind(this);
+    this.validateUsernamePassword = this.validateUsernamePassword.bind(this);
+  }
+
+  validateInput(req, res, next) {
+    if (!req.body.username || !req.body.password) {
+      this.getLogger().info('Missing required parameters.');
+      res.status(400).send('Missing required parameters.');
+    } else {
+      next();
+    }
+  }
+
+  validateUsernamePassword(req, res, next) {
```

Comment:
This is what a validator shouldn't do: deep logical validation (with a database).